scripts/common.py

```diff
@@ -649,6 +652,12 @@ def start_redis_server():
             'machine is on the Windows operating system. The redis server '
             'cannot start.')
 
+    # Check if a redis dump file currently exists. This file contains residual
+    # data from a previous run of the redis server. If it exists, removes the
+    # dump file so that the redis server starts with a clean slate.
+    if os.path.exists(REDIS_DUMP_PATH):
```

Comment:
Optional: Should we also remove this file in the clean stage i.e, around line 681? (not sure whether we should repeat this step in multiple places!) wdyt?

Reply:
I was thinking about removing it in the clean stage but I think that would invalidate the need to remove it here. I think it's cleaner just to remove it in one place and since Sean mentioned that making sure we should clean the server on startup, I added it to the startup script. If we removed it in startup and added it to cleanup, this would allow for other redis-server sets to possible corrupt the redis-server(if we start the server through the cli instead of through oppia and added a dump.rdb to the base directory that way). Cleaning it up at start is the most robust way of doing things I think. Or else there would be useless duplicate code.